third_party/3/pandas/core/frame.pyi

```diff
@@ -184,8 +184,15 @@ class DataFrame(NDFrame):
     def unstack(self, level: Union[Level, List[Level]] = ..., fill_value: Optional[Scalar] = ...) -> FrameOrSeries: ...
     def melt(self, id_vars: Optional[Union[Tuple[Column], List[Column], np.ndarray]] = ..., value_vars: Optional[Union[Sequence[Column], np.ndarray]] = ..., var_name: Optional[Scalar] = ..., value_name: Scalar = ..., col_level: Optional[Level] = ..., ignore_index: bool = ...) -> DataFrame: ...
     def diff(self, periods: int = ..., axis: AxisOption = ...) -> DataFrame: ...
+    # Less precise type to nudge folks towards `.agg`
```

Comment:
I am rather sceptical about this one. It would be awesome if people would use .agg rather than aggregate, but it shouldn't be indicated by this kind of tricks. It is not very obvious and we should have same signatures for aggregate and agg, in my opinion.

Reply:
I tend to agree. While it would be nice to promote `agg` over `aggregate`, user won't bother to check the stub's source code, when he runs into issues. Instead he might just decide to stop using them.